client/src/elm/Pages/WellChildProgressReport/View.elm

```diff
@@ -617,10 +617,7 @@ resolveDateOfLastNutritionAssessment currentDate isChw initiator mandatoryNutrit
 generatePartitionedWarningEntries :
     ModelIndexedDb
     -> Maybe AssembledData
-    ->
-        ( List ( NominalDate, PediatricCareMilestone, EncounterWarning )
-        , List ( NominalDate, PediatricCareMilestone, EncounterWarning )
-        )
+    -> ( List ( NominalDate, PediatricCareMilestone, EncounterWarning ), List ( NominalDate, PediatricCareMilestone, EncounterWarning ) )
```

Comment:
Why this was changed?

Reply:
> 

@anvmn just on save elm-format is changing it

Reply:
@Diama1 which version are yopu using?

Reply:
@anvmn Elm-format 0.8.3

Reply:
@Diama1 please update to `0.8.5`